package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "react-native-component-viewer",
-  "version": "0.6.1",
+  "version": "0.6.2",
```

Comment:
let's bump this to 0.7.0 instead as there is a breaking change with the peerDependencies